pulp_ansible/app/serializers.py

```diff
@@ -385,9 +385,14 @@ class TagSerializer(serializers.ModelSerializer):
     A serializer for the Tag model.
     """
 
+    count = serializers.SerializerMethodField()
+
+    def get_count(self, obj):
+        return obj.count
```

Comment:
I think it knows how to do that now... (Please try removing it).

You need to change the SerializerMethodField for something else...

Reply:
changed to integer field